MovieApp/PresentationLayer/MovieDetails/MovieHeader/CircularRating/CircularRatingView.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+
+class CircularRatingView: UIView {
+
+    let radius: CGFloat = 21
+    let lineThickness: CGFloat = 3
+
+    var circleLayer: CAShapeLayer!
+    var ratingLayer: CAShapeLayer!
+    var ratingLabel: UILabel!
+    var ratingContainer: UIView!
+    var numberLabel: UILabel!
+    var percentageLabel: UILabel!
+
+    override var intrinsicContentSize: CGSize {
+        CGSize(width: 2 * radius, height: 2 * radius)
+    }
+
+    init() {
+        super.init(frame: .zero)
+
+        buildViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setData(for rating: Double) {
+        guard rating >= 0, rating <= 10
```

Comment:
each condition in new line and extract 10 into field: let maxRating: Double = 10